common/app/views/fragments/atoms/storyQuiz.scala.html

```diff
@@ -0,0 +1,82 @@
+@import _root_.quiz.form._
+@import _root_.quiz.{postUrl, Question, Answer}
+@import views.html.fragments.inlineSvg
+@import layout.ContentWidths.BodyMedia
+@import views.support.RenderClasses
+
+@(quiz: model.content.StoryQuiz)
+
+<div id="@quiz.id" class="explainer-snippet explainer-snippet--light explainer-snippet--storyquiz storyquiz js-storyquiz">
+  @quiz.image.map { src => <img class="storyquiz__image" src="@src"> }
+    <div class="storyquiz__scene">
+      <div class="storyquiz__card storyquiz__card--intro is-active">
+        <div class="explainer-snippet__header">
+          <span class="explainer-snippet__label">Quick quiz</span>
+          <h4 class="explainer-snippet__headline">How much do you know about @quiz.title?</h4>
```

Comment:
Done. Fwiw, I'm not a huge fan of `fieldset` because UAs usually repeat the `legend` before reading every field. This can make things worse. I prefer using ARIA attributes over them.